templates/bundles/FOSUserBundle/Profile/edit.html.twig

```diff
@@ -0,0 +1,68 @@
+{% extends "bundles/FOSUserBundle/layout.html.twig" %}
```

Comment:
why keeping those templates as overridden FOSUserbundle templates (which are not accessed under the `@FOSUser` namespace anymore anyway) ?